src/data/customer/__test__/job.test.js

```diff
@@ -47,13 +52,15 @@ describe('Show job', async () => {
   });
 });
 
-describe('job create', async () => {
+describe('job create and delete test', async () => {
```

Comment:
Test is currently failing: https://travis-ci.org/CarPalFleet/JavaScript-SDK/builds/423733800?utm_source=github_status&utm_medium=notification (don't worry about the snapshots)